docs/topic-guides/kotlin/suggest.md

```diff
@@ -0,0 +1,85 @@
+# Suggest
+
+The API for the `SuggestStore` can be found in the [MozillaComponents Kotlin documentation](https://mozilla.github.io/application-services/kotlin/kotlin-components-docs/mozilla.appservices.suggest/-suggest-store/index.html).
+
+> Make sure you initialize [`viaduct`](../viaduct.md) for this component.
+
+> The `SuggestStore` is a synchronous, which needs to be wrapped in the asynchronous primitive of the target language you are using it in.
+
+## Setting up the store
+
+You need to import one or more of the following primitives to work with the `SuggestStore` (these come from the generated `suggest.kt` file, produced by `uniffi`):
+
+```kotlin
+import mozilla.appservices.remotesettings.RemoteSettingsServer
+import mozilla.appservices.suggest.SuggestApiException
+import mozilla.appservices.suggest.SuggestIngestionConstraints
+import mozilla.appservices.suggest.SuggestStore
+import mozilla.appservices.suggest.SuggestStoreBuilder
+import mozilla.appservices.suggest.Suggestion
+import mozilla.appservices.suggest.SuggestionQuery
+```
+
+Create a `SuggestStore` as a singleton. You do this via the `SuggestStoreBuilder`, which returns a `SuggestStore`.  No I/O or network requests are performed during construction, which makes this safe to do at any point in the application startup:
+
+```kotlin
+internal val store: SuggestStore = {
+    SuggestStoreBuilder()
+        .dataPath(context.getDatabasePath(DATABASE_NAME).absolutePath)
+        .remoteSettingsServer(remoteSettingsServer)
+        .build()
+```
+
+* You need to set the `dataPath`, which is the path (the SQLite location) where you store your suggestions.
+* The `remoteSettingsServer` is only needed if you want to set the server to anything else but `prod`. If so, you pass a `RemoteSettingsServer` object. 
+
+## Ingesting suggestions
+
+We have to handle different phases of the ingesting suggestions: The initial one on starting up the application, and then, periodically, in the background.
```

Comment:
```suggestion
Ingesting suggestions happens in two different ways: On startup, and then, periodically, in the background.
```

Here's my attempt to wordsmith this one, I'm open to other wordings as well.  The main things I'm concerned about are that "we" could be a bit confusing since there are multiple teams involved and also "phases" could imply that the application only needs to do one initial `emptyOnly=true` ingestion, but it should do it every startup to handle upgrades.  The reason for this is that a lot of times when we change the DB schema, we also throw away all the suggestions because it's easier than doing a real migration (and maybe we can't migrate the suggestions without data from remote settings).